azurerm/resource_arm_storage_blob.go

```diff
@@ -114,6 +114,14 @@ func resourceArmStorageBlob() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.IntAtLeast(1),
 			},
+
+			"metadata": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
```

Comment:
Should we validate this with `NoEmptyStrings`?

Reply:
I think that is a good call